src/Symfony/Component/Messenger/Bridge/Redis/Transport/Connection.php

```diff
@@ -203,13 +204,14 @@ public static function fromDsn(string $dsn, array $options = [], \Redis|\RedisCl
             };
         }
 
+        $pass = '' !== ($parsedUrl['pass'] ?? '') ? urldecode($parsedUrl['pass']) : null;
+        $user = '' !== ($parsedUrl['user'] ?? '') ? urldecode($parsedUrl['user']) : null;
+        // See: https://github.com/phpredis/phpredis/#auth
```

Comment:
the link is likely to age badly, I'd remove it

Reply:
I just moved it :) but I have removed it now - thanks.